tools/manifestfile.py

```diff
@@ -632,6 +635,8 @@ def main():
         exit(1)
 
     m = ManifestFile(mode, path_vars)
+    if args.unix_ffi:
+        m.add_library("unix-ffi", os.path.join("$(MPY_LIB_DIR)", "unix-ffi"))
```

Comment:
This should use `m.add_library(..., prepend=True)`.

Reply:
Oops, could have sworn I kept this. Good catch!